Add uninstall central components guide (#4970)
wazuh/wazuh-documentation: source/user-manual/uninstall/central-components.rst

.. Copyright (C) 2015-2022 Wazuh, Inc.

.. meta::
  :description: Learn how to uninstall each Wazuh central component.
  
Uninstalling the Wazuh central components
=========================================

You can uninstall all the Wazuh central components using the `Wazuh installation assistant <https://packages-dev.wazuh.com/|WAZUH_LATEST_MINOR|/wazuh-install.sh>`_.

Run the assistant with the option ``-u`` or ``--uninstall`` as follows:

    .. code-block:: console

      $ sudo bash wazuh-install.sh --uninstall

> This will remove Wazuh indexer, Wazuh server, and Wazuh dashboard from your cluster node.

Reviewer: ```suggestion
This will remove the Wazuh indexer, the Wazuh server, and the Wazuh dashboard.
```
